### DISCUSSION TITLE
There were some errors when starting the program

### DISCUSSION THREAD

**Discussion by iamnotandy** [community/community#70823] [Programming Help]
### Select Topic Area

Question

### Body

![屏幕截图 2023-10-22 214113](https://github.com/community/community/assets/130208229/28137fb2-0f4d-470f-8402-16c6c8179379)
![屏幕截图 2023-10-22 214250](https://github.com/community/community/assets/130208229/b049f4c8-d1e5-4454-bed5-77ccd9331a26)
hello everyone,i am a beginner.An error occurred when I was running a program called tinode on linux.I ask chatght then he told me there are some error in the code.Can someone tell me what I should do, please.THANK YOU VERY MUCH!!! 
project location : https://github.com/tinode

**Comment by Xrenya** [comment]
[Here](https://github.com/tinode/chat/blob/4184ed941080daec1bb6b7804f64fb6a4fc2818d/INSTALL.md) the installation and usage guide.

**Reply by iamnotandy** [reply]
thanks man.
